app/pages/nonsipp/landorproperty/WhyDoesSchemeHoldLandPropertyPage.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.landorproperty
+
+import models.SchemeHoldLandProperty
+import models.SchemeId.Srn
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case class WhyDoesSchemeHoldLandPropertyPage(srn: Srn) extends QuestionPage[SchemeHoldLandProperty] {
+
+  override def path: JsPath = JsPath \ toString
```

Comment:
Data path would be similar to the one commented on the ticket.
We have a similar example in the loans (with one element less), where saved off data looks like (key 0/1/2/3 are indexes of the elements - from the URL):
```
  "loans": {
    "loanTransactions": {
      "recipientIdentityType": {
        "whoReceivedLoans": {
          "0": "individual",
          "1": "ukCompany",
          "2": "ukPartnership",
          "3": "other"
        },
```